src/main.c

```diff
@@ -413,12 +440,6 @@ main(void)
         BLE_power(1, "Nano X");
 #endif
```

Comment:
If a Blue version of this app exists, then this BLE activation code block should be put *after* the `ui_idle()` call and *before* the `algorand_main()` call to prevent a bug on consumer Blue devices.
If the app doesn't build for Blue, then it doesn't matter.

Reply:
This app is not meant to run on Blue devices.